main.js

```diff
@@ -42,10 +43,17 @@ module.exports = (glob, options) => {
     let mainFile = path.join(appPath, config.main);
 
     chokidar.watch(mainFile).on('change', () => {
-      proc.spawn(eXecutable, [appPath], {
-        detached: true,
-        stdio: 'inherit',
-      });
+	  // Detaching child is useful when in Windows to let child
+	  // live after the parent is killed
+	  if (isWin) {
```

Comment:
Please remove the `isWin` condition, it disturbs the readability :+1: . it works also fine under linux.